lib/node/nodes/point-in-polygon.js

```diff
@@ -1,45 +0,0 @@
-'use strict';
```

Comment:
We should not remove existing analyses, otherwise we will break some applications.

I know this is not 1.x version but I prefer to keep them all for now.